cf-smoke/pom.xml

```diff
@@ -85,6 +85,15 @@
                 </executions>
             </plugin>
         </plugins>
+
+        <extensions>
+            <!-- Upload the artifacts to an azure blob store container -->
+            <extension>
+                <groupId>com.gkatzioura.maven.cloud</groupId>
+                <artifactId>azure-storage-wagon</artifactId>
```

Comment:
do we need this ?

Reply:
We do have it on the smoke project and do need it in case we want to deploy the artifacts to the blob store instead of Artifactory.
As we have the snapshot repository only on blob store, I added this so that I can easily deploy SNAPSHOT versions for testing.

I am fine to remove it but let it in as the same was already there in the public pom.xml for smoke:
https://github.com/adobe/aem-test-samples/blob/aem-cloud/smoke/pom.xml#L89-L96

Deploy command which we can use to build snapshots and use them in EaaS

```
mvn deploy -DaltSnapshotDeploymentRepository=azure-eaastests::default::bs://eaastests/maven-eaas-snapshot
```

Reply:
got it, fine for me